File: src/main/java/plugin/command/owner/RemoveNPCSpawnCommandPlugin.java
package plugin.command.owner;

import org.redrune.game.entity.actor.player.Player;
import org.redrune.game.content.plugin.type.CommandPlugin;

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 9/4/2017
 */
public class RemoveNPCSpawnCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
			player.putTemporaryAttribute("removing_npcs", !player.getTemporaryAttribute("removing_npcs", false));
			player.getPackets().sendMessage("You are now " + (player.getTemporaryAttribute("removing_npcs", false) ? "removing" : "examining") + " npcs.");
	}
	
	@Override
	public String[] identifiers() {
		return arguments("rspns");
	}
}

File: src/main/java/plugin/command/owner/RemoveNPCSpawnCommandPlugin.kt
package plugin.command.owner

import game.content.plugin.type.CommandPlugin
import game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur></itstyluur>@icloud.com>
 * @since 9/4/2017
 */
class RemoveNPCSpawnCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        player.putTemporaryAttribute("removing_npcs", !player.getTemporaryAttribute("removing_npcs", false))
        player.packets.sendMessage(
            "You are now " + (if (player.getTemporaryAttribute(
                    "removing_npcs",
                    false
                )
            ) "removing" else "examining") + " npcs."
        )
    }

    override fun identifiers(): Array<String> {
        return arguments("rspns")
    }
}